__tests__/index.test.js

```diff
@@ -0,0 +1,127 @@
+const postcss = require("postcss");
+
+jest.mock("fs", () => {
+  return {
+    writeFile: jest
+      .fn()
+      .mockImplementation((file, path, cb) => cb("foo"))
+      .mockName("nodeWriteFile"),
+  };
+});
+
+const plugin = require("../index");
+
+async function run(input, opts = {}) {
+  return await postcss([plugin(opts)]).process(input, {
+    from: undefined,
+  });
+}
+
+describe("postcss-validate-customprop", () => {
+  afterAll(() => {
+    jest.resetModules();
+  });
+
+  test("validates CSS", async () => {
```

Comment:
@abhinavpreetu `validates custom properties fallback values` would be more accurate?